media/src/activity/activity-map.tsx

```diff
@@ -50,7 +53,7 @@ export interface ResponseData {
     submitted_at: Date;
     modified_at: Date;
     submitted_at_formatted: string;
-    modified_at_formatted: string;
+    modified_at_formatted?: string;
```

Comment:
Why is there a question mark here?

Reply:
That's a good question.

It makes the attribute optional, meaning that its value could be a string or it could be null. This makes TypeScript signal to the programmer that you need to check for null before it is referenced.

Seeing your question though makes me realize its unnecessary. A modified time would be set at creation. I'll remove this.